src/visit/mod.rs

```diff
@@ -540,28 +540,6 @@ pub trait VisitMap<N> {
     fn is_visited(&self, a: &N) -> bool;
 }
 
-impl<Ix> VisitMap<graph::NodeIndex<Ix>> for FixedBitSet
-    where Ix: IndexType,
-{
-    fn visit(&mut self, x: graph::NodeIndex<Ix>) -> bool {
-        !self.put(x.index())
-    }
-    fn is_visited(&self, x: &graph::NodeIndex<Ix>) -> bool {
-        self.contains(x.index())
-    }
-}
-
-impl<Ix> VisitMap<graph::EdgeIndex<Ix>> for FixedBitSet
-    where Ix: IndexType,
-{
-    fn visit(&mut self, x: graph::EdgeIndex<Ix>) -> bool {
-        !self.put(x.index())
-    }
-    fn is_visited(&self, x: &graph::EdgeIndex<Ix>) -> bool {
-        self.contains(x.index())
-    }
-}
-
```

Comment:
They became unnecessary (and conflicting) because of https://github.com/petgraph/petgraph/commit/30fb481d7971732b1a025acb1c5b8ecc697b0fef#diff-ad1a163e094362a54511320c14281ffc